src/Raven.Studio/typescript/models/database/tasks/exportDatabaseModel.ts

```diff
@@ -69,8 +74,14 @@ class exportDatabaseModel {
 
     private initValidation() {
         this.exportDefinitionHasIncludes = ko.pureComputed(() => {
-            return this.includeDatabaseRecord() || this.includeDocuments() || (this.includeRevisionDocuments() && this.revisionsAreConfigured()) || this.includeConflicts() ||
-                this.includeIndexes() || this.includeIdentities() || this.includeCompareExchange() || this.includeCounters();
+            if (this.includeAttachments()) {
+                this.validationMessage("Note: Attachments can only be included with documents.");
```

Comment:
Can we automatically check documents when user checks attachments + avoid having generic field: validationMessage in this model?